README.md

```diff
@@ -69,46 +69,147 @@ fields that can be selected, so implementations must provide this out-of-band. I
 fields in Item Properties to be prefixed with `properties.` or not, or support use of both the prefixed and non-prefixed
 name, e.g., `properties.datetime` or `datetime`.
 
-## Include/Exclude Semantics 
+## Include/Exclude Semantics
 
-1. If `fields` attribute is specified with an empty object, or with both `include` and `exclude` set to null or an 
-empty array, the recommended behavior is as if `include` was set to 
-`["id", "type", "geometry", "bbox", "links", "assets", "properties.datetime"]`.  This default is so that the entity 
-returned is a valid STAC Item.  Implementations may choose to add other properties, e.g., `created`, but the number 
+1. If `fields` attribute is specified as an empty object (for POST requests), an empty string (for GET requests),
+or with both `include` and `exclude` set to null or an
+empty array, then the recommended behavior is to include only fields
+`type`, `stac_version`, `id`, `geometry`, `bbox`, `links`, `assets`, and `properties.datetime`.
+This default is so that the entity returned is a valid STAC Item.
+Implementations may choose to include other properties, e.g., `properties.created`, but the number
 of default properties attributes should be kept to a minimum.
-2. If only `include` is specified, these attributes are added to the default set of attributes (set union operation). 
-3. If only `exclude` is specified, these attributes are subtracted from the union of the default set of attributes and 
-the `include` attributes (set difference operation).  This will result in an entity that is not a valid Item if any 
-of the excluded attributes are in the default set of attributes.
-4. If both `include` and `exclude` attributes are specified, semantics are that a field must be included and **not** 
-excluded.  E.g., if `properties` is included and `properties.datetime` is excluded, then `datetime` must not appear 
-in the attributes of `properties`.
+2. If only `include` is specified, these attributes should be the only attributes included.
+If additional fields are provided beyond those in the `include` list, the number and size of
+these attributes should be kept to a minimum.
+3. If only `exclude` is specified, these attributes fields should not be
+included in the response Item object, but every other fields available for the
+Item should be included.
+4. If `exclude` is specified and `include` is null or an empty
+array, then the `exclude` attributes should be excluded from the default set.
+5. For nested attributes (e.g. `properties.datetime`), the most specific path
```

Comment:
let's change all the uses of `attribute` to `field` -- I think it's confusing to have both.